packages/next/next-server/server/load-components.ts

```diff
@@ -21,6 +21,7 @@ export type LoadComponentsReturnType = {
     props: any
     revalidate: number | false
   }
+  unstable_getStaticParams?: () => void
```

Comment:
The type is incorrect right?

Something like this would be correct:

```suggestion
  unstable_getStaticParams?: () => Array<{[string]: string} | string>
```

Reply:
I didn't type it since it's likely to change, it's only being used to check if it exists right now

Reply:
Okay!